src/main/webapp/app/exercises/quiz/shared/quiz-manage-util.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { QuizExercise, QuizMode, QuizStatus } from 'app/entities/quiz/quiz-exercise.model';
+
+@Injectable({ providedIn: 'root' })
+export class QuizManageUtil {
+    /**
+     * Check if quiz is editable
+     * @param quizExercise the quiz exercise which will be checked
+     * @return {boolean} true if the quiz is editable and false otherwise
+     */
+    isQuizEditable(quizExercise: QuizExercise): boolean {
```

Comment:
Is it really necessary to create such a class with root injection? The method could also be simply static and be imported from a util typescript file

Reply:
I followed ShortAnswerUtil class. Thanks for the suggestion! I changed this to an exported function.